.github/workflows/ruby.yml

```diff
@@ -22,27 +22,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.3, 2.4, 2.5, 2.6, 2.7]
-        rails: ["4.1", "4.2", "5.0", "5.2.3", "6.0"]
-        exclude:
-          - ruby: "2.4"
-            rails: "4.1"
-          - ruby: "2.4"
-            rails: "4.2"
-          - ruby: "2.4"
-            rails: "6.0"
-          - ruby: "2.5"
-            rails: "4.1"
-          - ruby: "2.5"
-            rails: "4.2"
-          - ruby: "2.6"
-            rails: "4.1"
-          - ruby: "2.6"
-            rails: "4.2"
-          - ruby: "2.7"
-            rails: "4.1"
-          - ruby: "2.7"
-            rails: "4.2"
+        ruby:
+          - "3.0"
```

Comment:
Needs to be quoted, otherwise `3.0` becomes `3`, which gets interpreted as `3.x`, resolving to `3.3`